Contest_Package.md

```diff
@@ -63,6 +63,37 @@ and `contest/<filename>` respectively, and files referenced in
 Note that the API specification requires that filenames are unique within
 endpoint objects, so this is always possible.
 
+#### Default filenames
+
+In some situations you may want file references to be found on disk
+without directly specifying them in a json file, for instance to keep
+json files simple or when providing organization logos or team photos
+manually or from different systems. Files that use the default
+filename pattern must be loaded in addition to any file references
+found in the json files.
+
+The default filename pattern is
+`<endpoint>/<id>/<property>(.variant).<extension>`, where:
+
+- `<id>` is the ID of the endpoint object the reference is in.
+- `<property>` is the property of the object, e.g. `logo`.
+- `(.<variant>)` is an optional variant. Must be used when there are
+  multiple files to avoid collisions.
+- `<extension>` is a file extension corresponding to the mime type.
+
+For images, the supported file extensions are:
+
+| Mime type     | Extension(s)
+| ------------- | ---------
+| image/png     | png
+| image/jpeg    | jpg, jpeg
+| image/svg+xml | svg
+
+For images, the variant should be the width x height of the image,
+e.g. logo.56x56.png.
```

Comment:
Aren't right now the width x height separate properties? So I'd change this to something like:
```suggestion
For images, the property (e.g. `logo`) should be followed by `.<W>x<H>` where W and H are the width and height of the image in pixels, then followed by any optional variants,
e.g. `logo.56x56.light.png`.
```